docs/rag.md

```diff
@@ -151,33 +227,27 @@ export const indexPdf = defineFlow(
 
 async function extractTextFromPdf(filePath: string) {
   const pdfFile = path.resolve(filePath);
-  const dataBuffer = fs.readFileSync(pdfFile);
+  const dataBuffer = await readFile(pdfFile);
   const data = await pdf(dataBuffer);
   return data.text;
 }
 ```
 
-To run the flow:
+#### Run the indexer flow
 
 ```posix-terminal
 genkit flow:run indexPdf "'../pdfs'"
 ```
 
-## Retrievers
+The local database now has documents stored in it!
```

Comment:
The reader can't actually follow along since the PDFs don't exist for real. Maybe rephrase accordingly.

Reply:
done.